models/setup_for_test.go

```diff
@@ -30,8 +30,8 @@ func TestMain(m *testing.M) {
 	setting.RunUser = "runuser"
 	setting.SSH.Port = 3000
 	setting.SSH.Domain = "try.gitea.io"
-	setting.RepoRootPath = "/repos"
-	setting.AppDataPath = "/appdata"
+	setting.RepoRootPath = "/tmp/repos"
```

Comment:
This certainty won't work on Windows. We should use [os.TempDir](https://golang.org/pkg/os/#TempDir) instead.

```go
filepath.Join(is.TempDir(), "repos")
```